### DISCUSSION TITLE
Can I handle audio and video without starting the stream.

### DISCUSSION THREAD

**Discussion by sanazaheer1400** [ant-media/Ant-Media-Server#5698] [General]
My question is that I want to check that audio and video working fine or not before starting the stream, should I do that with Android SDK. I checked in sdk I checked only when I start the stream.

**Comment by tremzo** [comment]
@sanazaheer1400 as I understand you want to test or preview your video and audio before you live stream to an audience, is that correct? There isn't any such feature right now, I can suggest you just broadcast to a test stream to create a preview.
